tasks/kernels/run.py

```diff
@@ -107,14 +99,41 @@ def log_2(x):
 
 
 def _process_kernels_result(
-    kernels_out, result_file, kernel, np, run_num, real_time
+    result_txt, result_file, kernel, np, run_num, measured_time=None
 ):
     stats = PRK_STATS.get(kernel)
 
     if not stats:
         print("No stats for {}".format(kernel))
         return
 
+    # Prepare response output and elapsed time
+    if "wasm" in result_file:
+        try:
+            result_json = json.loads(result_txt, strict=False)
+        except json.decoder.JSONDecodeError:
+            print(
+                "WARNING: can't process result json for {}, {} (run {})".format(
+                    kernel, np, run_num
+                )
+            )
+            pprint(result_txt)
+            return
```

Comment:
When does this error occur? Do we need to guard against it and handle it silently? Seems to me it would be worth stopping the experiment (i.e. not catching the exception).

Reply:
I thought we'd rather be resilient to one-off failures that cancel the whole experiment, specially for long experiments.

However, we do already bail out if the return code is not `SUCCESS`, thus it is not clear when we'd succeed yet not return a valid JSON.

I will remove the expection guard completely.